ui/Rhino/AGS/dev/AGS_form_deviation_threshold_cmd.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import scriptcontext as sc
+
+import compas_rhino
+
+
+__commandname__ = "AGS_form_deviation_threshold"
+
+
+def RunCommand(is_interactive):
```

Comment:
why is this not just part of the global settings dialog?

Reply:
Hi tom. This is part of the general dialog! Therefore there's no way how this is editable via mac. So this function and the toggle_autoupdate are there to bring it to mac. You have other suggestion?

Reply:
there should be a general settings form for Mac as well. RV2 has one...

Reply:
ok, so should I remove this from the PR, and open an issue in the issue tracker?

Reply:
Or you think we can accept it and open the issue to deal with it in the future? I would say that it is annoying currently not being able to change these values from AGS... does this pass as a temporary solution?